components/Common/CodeBox/CodeTabs/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+import type { FC, PropsWithChildren } from 'react';
+
+import type { TabsProps } from '@/components/Common/Tabs';
+import Tabs from '@/components/Common/Tabs';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type CodeTabsProps = Pick<
+  TabsProps,
+  'tabs' | 'onValueChange' | 'defaultValue'
+> & {
```

Comment:
I'd rather than addons could be just `children`? So we can simply add what we want?

Then you can create simply update the `LocalizedLink` component to a have a new prop called `external` (boolean) and when present it renders the Arrow.

Then we simply pass on `<CodeTabsProps><LocalizedLink external={true}>....`

Reply:
If this is about the `addons` added in the `Tabs` component, then we can't just render the link as a child because the level is different. 

In order to respect this layout:

![image](https://github.com/nodejs/nodejs.org/assets/23018799/55666f1f-828c-4599-a8e5-2bcb8416e6ae)

We need to render the link inside the `TabsPrimitive.List`. This way it is possible to use a flex layout and harmonize the tabs with the link.

![image](https://github.com/nodejs/nodejs.org/assets/23018799/8c1c998c-4cf5-4d9f-be87-0647d6f1a19f)

Do you think of something else?

Reply:
I see... Well it's fine then.